src/python/pants/backend/python/goals/setup_py.py

```diff
@@ -764,6 +768,187 @@ def is_ownable_target(tgt: Target, union_membership: UnionMembership) -> bool:
     )
 
 
+# Convenient type alias for the pair (package name, data files in the package).
```

Comment:
`rules/util.py` no longer really fit anywhere. It's not relevant to `util_rules/`. Turns out, 90% of the contents are specific to `setup_py.py`, and the other 10% are specific to `pex.py`. So, I inlined into those two files.